### ISSUE TITLE
Strange dots while using Pedestrian Mode only

### ISSUE THREAD

**Issue by iphonic** [mapsme/omim#10917]
Hi, I am facing an issue when using **Pedestrian Mode** the map strangely draw green dots at numbers, if I close the current routing and start to get a new route of new locations, the previous dot remains there and more dots are plotted.

I am adding a video of the behaviour see there https://www.screencast.com/t/SWod725Qds it only occurs in Walk Mode.

If anyone could point me from where I can turn off this drawing it would be very helpful.

Thanks.

**Comment by gmoryes** [comment]
Hello, this is not maps.me, It someone's fork of our application.

**Comment by gmoryes** [comment]
maps.me looks like:
![image](https://user-images.githubusercontent.com/17534533/58046426-e9fed100-7b4d-11e9-85e3-ac38a9290766.png)

**Comment by gmoryes** [comment]
You can download the original app here: https://play.google.com/store/apps/details?id=com.mapswithme.maps.pro&hl=ru

**Comment by iphonic** [comment]
Yes, it is maps.me but not the latest version it uses version **161010**. Due to several reasons it can't be updated to latest yet, it would be very helpful if any pointer is given as the core framework of maps.me untouched. 

Thanks.

**Comment by gmoryes** [comment]
> Yes, it is maps.me but not the latest version it uses version **161010**. Due to several reasons it can't be updated to latest yet, it would be very helpful if any pointer is given as the core framework of maps.me untouched.
> 
> Thanks.

I'm repeating - this is not original maps.me application, it someone's clone.

You can download the origin app here: [for android in google play](https://play.google.com/store/apps/details?id=com.mapswithme.maps.pro&hl=ru), [and for ios in app store](https://itunes.apple.com/us/app/maps-me-offline-map-nav/id510623322?mt=8)
